recipes/py2neo/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name='py2neo' %} 
+{% set version='4.0.0b2' %}
+{% set md5='d7ff00b7af97241f104be88b30cd2e45' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/97/3c/6257962dd23ad791d6e885116b734e55602980c862b028e684b38cc1d96c/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  patches:
+    - console.patch
```

Comment:
Can you explain why this patch is needed in a comment here? Also add a link if you got this patch from somewhere.